Lib/dis.py

```diff
@@ -536,6 +536,16 @@ def dis(self):
                                lasti=offset)
             return output.getvalue()
 
+    def __eq__(self, other):
```

Comment:
I didn't add a check about underlying code object because I think we should compare the bytecode not the source but I'm OK to change it if needed?
```py
>>> dis.Bytecode("print(1)") == dis.Bytecode(compile("print(1)", "<stdin>", "eval"))
True
>>> dis.Bytecode("print(1)")._original_object == dis.Bytecode(compile("print(1)", "<stdin>", "eval"))._original_object
False
```